app/build.gradle

```diff
@@ -26,7 +26,7 @@ android {
 }
 
 dependencies {
-    implementation "com.ifttt:connect-button:2.0.0"
+    implementation project(':connect-button')
```

Comment:
Intentional change: this is to allow us to debug and make changes using Grocery Express and the Connect Button module.